content/ko/docs/concepts/overview/working-with-objects/annotations.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 어노테이션
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+쿠버네티스 어노테이션을 사용하여 임의의 비-식별 메타데이터를
+오브젝트에 첨부할 수 있다. 도구 및 라이브러리와 같은 클라이언트는 이 메타데이터를 검색할 수 있다.
+{{% /capture %}}
+
+{{% capture body %}}
+## 오브젝트에 메타데이터 첨부
+
+레이블이나 어노테이션을 사용하여 쿠버네티스
+오브젝트에 메타데이터를 첨부할 수 있다. 레이블을 사용하여 오브젝트를 선택하고, 특정 조건을 만족하는 오브젝트
+컬렉션을 찾을 수 있다. 반면에, 어노테이션은
+오브젝트를 식별하고 선택하는데 사용되지 않는다. 어노테이션의 메타데이터는
+작거나 크거나, 구조적이거나 구조적이지 않을 수 있으며, 레이블에서
+허용되지 않는 문자를 포함할 수 있다.
+
+어노테이션은 레이블과 같이 키/값 맵이다.
+
+```json
+"metadata": {
+  "annotations": {
+    "key1" : "value1",
+    "key2" : "value2"
+  }
+}
+```
+
+다음은 어노테이션에 기록할 수 있는 정보의 예제이다.
+
+* 필드는 선언적 구성 계층에 의해 관리된다. 이러한 필드를 어노테이션으로 첨부하면
+  클라이언트 또는 서버가 설정한 기본 값과,
+  자동 생성된 필드, 그리고
+  오토사이징 또는 오토스케일링 시스템에서 설정된 필드와 구분된다.
+
+* 빌드, 릴리스, 또는 타임 스탬프, 릴리즈 ID, git 브랜치,
+  PR 번호, 이미지 해시 및 레지스트리 주소와 같은 이미지 정보
+
+* 로깅, 모니터링, 분석 또는 감사 리포지토리에 대한 포인터.
+
+* 디버깅 목적으로 사용될 수 있는 클라이언트 라이브러리 또는 도구 정보:
+  예를 들면, 이름, 버전, 그리고 빌드 정보
+
+* 다른 생태계 구성 요소의 관련 오브젝트 URL과 같은
+  사용자 또는 도구 / 시스템 출처 정보.
+
+* 경량 롤아웃 도구 메타데이터. 예: 구성 또는 체크포인트
+
+* 책임자의 전화번호 또는 호출기 번호, 또는 팀 웹 사이트 같은
+  해당 정보를 찾을 수 있는 디렉토리 진입점.
+
+* 행동을 수정하거나 비표준 기능을 수행하기 위한
+  최종 사용자의 지시 사항.
+
+어노테이션을 사용하는 대신, 이 유형의 정보를
+외부 데이터베이스 또는 디렉토리에 저장할 수 있지만, 배포, 관리, 내성(introspection) 등과 같은
+공유 클라이언트 라이브러리와 도구를 만드는 것이
```

Comment:
어노테이션을 사용하는 대신, 이 유형의 정보를
외부 데이터베이스 또는 디렉토리에 저장할 수 있지만, 배포, 관리, 내성(introspection) 등과 같은
공유 클라이언트 라이브러리와 도구를 만드는 것이
훨씬 어려울 수 있다.

[원문]
Instead of using annotations, you could store this type of information in an
external database or directory, but that would make it much harder to produce
shared client libraries and tools for deployment, management, introspection,
and the like.

[제안]
어노테이션을 사용하는 대신, 이 유형의 정보를
외부 데이터베이스 또는 디렉토리에 저장할 수 있지만, **이는** 배포, 관리, 인트로스펙션(introspection) 등을 **위한**
공유 클라이언트 라이브러리와 도구 **생성을**
훨씬 **더 어렵게 만들 수 있다.**

introspection를 내성(introspection)으로 번역하셨는데, 여기서 introspection은 "내성", "자기성찰" 등의 일반적인 의미로 쓰였다기보다는 프로그램 내의 속성이나 오브젝트를 런타임에서 시험하기 위한 태스크와 같이 기술적인 의미로 쓰인 것 같습니다.
[In computing, type introspection is the ability of a program to examine the type or properties of an object at runtime. Some programming languages possess this capability.](https://en.wikipedia.org/wiki/Type_introspection)

회의에서 논의가 필요해 보이지만, 우선 "인트로스펙션(introspection)"으로 제안 드립니다.